src/vm/Components/FileHandlers/Writer.java

```diff
@@ -0,0 +1,27 @@
+package vm.Components.FileHandlers;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class Writer {
+
+  private String filename;
+  private Path path;
+
+  public Writer(String filename) {
+    Path path = Path.of("src", "resources", filename);
+    this.path = path;
+    this.filename = filename;
+  }
+
+  public void write(String arg) {
+    try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(path.toAbsolutePath().toString(), true))) { 
+      bufferedWriter.write(arg);
+    } catch(IOException e) {
```

Comment:
Não precisa fazer um catch de `FileNotFoundException` aqui também?
Talvez mandar essa criação do bufferedWriter pro construtor, como foi feito no Reader, pra manter o padrão, e já que se o arquivo não existir não teria como usar a classe mesmo.

Reply:
o catch de FileNotFoundException não é necessário porque o método FileWriter joga IOException e ela já engloba a outra exceção. Como a gente não ta fazendo nada de fato com a exceção atualmente além de printar o stack trace, não acho que a gente precise pegar a exceção mais específica se a gente ja é obrigado a lançar a qual ela herda

Reply:
Boa!!
Ahh e não sabia que FileWrite não jogava a exceção FileNotFoundException especificamente, valeu por explicar!

Reply:
resolved in commits 4a4d5d139ce1b5808595032ac1aa427c25991afa and 2b5998d8ab3e3fbe268d098a35c8fc2bdce4c5cb